src/components/Editor.jsx

```diff
@@ -154,19 +203,29 @@ export default function Editor( props) {
     blockable,
     editable,
     preview,
+    allAligned: (!brokenAlignedWords || brokenAlignedWords.length===0),
+    onShowUnaligned: handleUnalignedClick,
     undo,
     redo,
     canUndo,
     canRedo,
     setToggles,
     canSave,
-    onSave:handleSave,
+    onSave: handleSave,
     showToggles:false
```

Comment:
This is the place where the first few Editor buttons are hidden - change this to true in order to reveal them